Adyen/Helpers/UIViewConstraintsHelper.swift

```diff
@@ -13,27 +13,31 @@ extension AdyenScope where Base: UIView {
     /// Attach top, bottom, left and right anchores of this view to corespondent anchores inside specified view.
     /// IMPORTANT: both view should be in the same hierarcy.
     /// - Parameter view: Container view
-    public func anchore(inside view: UIView, with padding: UIEdgeInsets = .zero) {
+    @discardableResult
+    public func anchore(inside view: UIView, with padding: UIEdgeInsets = .zero) -> [NSLayoutConstraint] {
```

Comment:
i think it is misspelled, should be `anchor` (both a noun and verb). probably multiple places in the project

Reply:
🙀

Reply:
PR for it [here](https://github.com/Adyen/adyen-ios/pull/448)